app/controllers/applications_controller.rb

```diff
@@ -16,12 +16,14 @@ def create
       redirect_to application_path(application.id)
     else
       flash[:danger] = current_user.errors.full_messages
+      @user = current_user
       render :new
     end
   end
 
   def show
     @amount = @application.user.domestic_applicant? ? 50 : 75;
+    @user = User.find_by(id: current_user.id)
```

Comment:
Change to `@user = current_user` - you already have that user - no need to do another db lookup!